src/Controller/Component/CrudComponent.php

```diff
@@ -649,7 +650,12 @@ protected function _loadListener($name)
                 throw new ListenerNotConfiguredException(sprintf('Listener "%s" is not configured', $name));
             }
 
+            if (!isset($config['className'])) {
```

Comment:
i would expect us to defer a class name like cake does based on the $name if no className is set